ci_framework/roles/test_deps/tasks/main.yml

```diff
@@ -71,6 +71,7 @@
     cifmw_repo_setup_output: "/etc/yum.repos.d"
     cifmw_repo_setup_branch: "master"
     cifmw_repo_setup_promotion: "current-podified"
+    cifmw_repo_setup_os_release: "centos"
```

Comment:
I have the feeling this one is either not part of this PR, or there's some missing details... ?

Reply:
OK - it's in the actual commit message, not in the first comment:
```
Note: Set cifmw_repo_setup_os_release to centos for ubi
```

Reply:
Yes, Updated in commit message also.